### ISSUE TITLE
How do I edit the localhost IP target?

### ISSUE THREAD

**Issue by K1nesis** [ronancpl/HeavenMS#437]
I followed this tutorial and did everything except use HEX-EDIT "localhost.exe" to fetch my IP.

As I searched for "Text String" "127.0.0.1",like this:
![image](https://user-images.githubusercontent.com/25972288/54632695-a63e0b80-4ab9-11e9-90ab-9a0372fa9cca.png)

I got the result:
![image](https://user-images.githubusercontent.com/25972288/54632648-87d81000-4ab9-11e9-8ac0-8a2a536ab990.png)

I have no experience in reverse engineering.

I don't know what I did wrong.

So can you provide a more detailed tutorial about this ?

if you can add some screenshots it will be awesome !

Thanks a lot.

**Comment by ryantpayton** [comment]
I believe you want to search by Text or String, can't remember the exact wording, and make sure you try "search up" and "search down". Otherwise you probably have a bad client. Are you using the client provided by the readme?

**Comment by K1nesis** [comment]
Thank you for your reply,but I can't speak Japanese.
I'm sure the client I used was provided by readme.   
Link: https://hostr.co/tsYsQzzV6xT0
And I've tried  "search up" and "search down", but not found. 
I'm using the VPS IP instead of "localhost" as the target IP on the server's config file. 
So I want to know how to use the Neo Hex Editor to modify the address in "localhost.exe".

我会说中文，不过你可以发日语给我，我用软件来翻译。

**Comment by ronancpl** [comment]
@K1nesis On field Type select "String", then try the Find again. It should get you to the 3 IP areas.

**Comment by K1nesis** [comment]
@ronancpl 
You mean this ? 
![image](https://user-images.githubusercontent.com/25972288/54659424-b9c69200-4b0c-11e9-9762-f9aefa161168.png)

I got a lot of results, not three :
![image](https://user-images.githubusercontent.com/25972288/54659611-8801fb00-4b0d-11e9-8f4d-f2648670629c.png)

**Comment by jaydenseah** [comment]
Try changing your system locale to English

**Comment by K1nesis** [comment]
@jaydenseah 
I've change my system locale to English.
![image](https://user-images.githubusercontent.com/25972288/54665187-3fecd380-4b21-11e9-98f8-b8a179a81d3c.png)

Now,it became like this:
![image](https://user-images.githubusercontent.com/25972288/54664663-dcae7180-4b1f-11e9-8121-97e2fbdde893.png)

Is this the right result? 
And i can find lots of "IP",here are some examples :
![image](https://user-images.githubusercontent.com/25972288/54664981-b0472500-4b20-11e9-9f3e-452e2db0d228.png)
![image](https://user-images.githubusercontent.com/25972288/54664989-b806c980-4b20-11e9-9bfe-f9ecf7d9ca74.png)
![image](https://user-images.githubusercontent.com/25972288/54664998-bf2dd780-4b20-11e9-8384-0808f778b3d1.png)

Which one should I edit?

**Comment by jaydenseah** [comment]
You should search 127.0.0.1 not IP...

**Comment by K1nesis** [comment]
@jaydenseah 
oh....

Can you give me a screenshot?

I don't know how to search 127.0.0.1, using hex or Sring?
![image](https://user-images.githubusercontent.com/25972288/54665538-59dae600-4b22-11e9-8b87-1695b2c0095e.png)
The result is not found.

**Comment by K1nesis** [comment]
Oh, I found I ignored one point.

Should i edit this "localhost.exe":
![image](https://user-images.githubusercontent.com/25972288/54668876-ef7a7380-4b2a-11e9-9a7d-2f372db66e6c.png)
or this one:
![image](https://user-images.githubusercontent.com/25972288/54668913-0751f780-4b2b-11e9-8217-967a4de6e857.png)

I have been trying to edit"MapleStory.exe" before.

Now, I edit the "ManagerMsv83.exe" by this settings : 
![image](https://user-images.githubusercontent.com/25972288/54669496-7b40cf80-4b2c-11e9-8322-be5fcd491862.png)

And i get 11 result in file,like this:
![image](https://user-images.githubusercontent.com/25972288/54669626-cfe44a80-4b2c-11e9-8554-3a8e59b1b218.png)

Is this the right result?

I hope to get some advice.

Thanks for all guys.

**Comment by K1nesis** [comment]
@ryantpayton @ronancpl @jaydenseah 
I made a big mistake.
The The correct file should be this:
![image](https://user-images.githubusercontent.com/25972288/54672193-872f9000-4b32-11e9-99f1-c6323787e545.png)

I have found the address i want to edit:
![image](https://user-images.githubusercontent.com/25972288/54672372-d70e5700-4b32-11e9-9498-6dd8a131010e.png)

Thanks all guys and your patient reply!

At the same time, I'm sorry because my stupid mistakes waste everyone's time.

I will try to edit it later. If there is no problem, I will close it.

Thanks everyone again.
